samples/OneClickDemo/OneClickDemo-Katalog.sh

```diff
@@ -264,9 +264,10 @@ export SECRET_KEY=1234
 export ENDPOINT="http://127.0.0.1:4566"
 export BUCKET="demo"
 export OBJECT_KEY="sample.csv"
-bin/aws configure set aws_access_key_id ${ACCESS_KEY} 
-bin/aws configure set aws_secret_access_key ${SECRET_KEY}
-bin/aws --endpoint-url=${ENDPOINT} s3api create-bucket --bucket ${BUCKET}
+export REGION=theshire
+bin/aws configure set aws_access_key_id ${ACCESS_KEY} && aws configure set aws_secret_access_key ${SECRET_KEY}
```

Comment:
can you separate these 2 commands in different lines.

Reply:
Done